charts/stable/zalando-postgres-cluster/values.yaml

```diff
@@ -77,7 +77,8 @@ dumpBackup:
       memory: "10Mi"
       cpu: "5m"
 
-# Dissable Deployment and Service
+# Dissable Deployment and Service generation
 controllerType: ""
```

Comment:
`controllerType` is not a supported field.

Reply:
This one is a problem: the common v3 does not allow to run without controller. Will ping you to discuss a possible PR to add.

Reply:
solved with 3.1